modules/seqexec/server/src/main/scala/seqexec/server/ObserveActions.scala

```diff
@@ -124,7 +127,8 @@ trait ObserveActions {
       _ <- notifyObserveStart(env)
       _ <- env.headers(env.ctx).traverse(_.sendBefore(env.obsId, fileId))
       _ <- info(s"Start ${env.inst.resource.show} observation ${env.obsId.format} with label $fileId")
-      r <- env.inst.observe(env.config)(fileId)
+      t <- env.inst.calcObserveTime(env.config)
+      r <- env.inst.observe(env.config)(fileId).timeout((1.5 * t.toMilliseconds).milliseconds)
```

Comment:
The 1.5 is fairly arbitrary. maybe it should be a constant like 1 min. The idea being that we cannot wait exactly obs time as there are overheads, etc but don't want to wait to long to declare it failed